src/components/elements/carousel/carousel-style.ts

```diff
@@ -1,9 +1,27 @@
 import React from 'react'
 import styled, { css } from 'styled-components'
+import type { FlattenSimpleInterpolation } from 'styled-components'
 import { Flex } from 'components/containers'
 import device from 'themes/device'
 import { ReactComponent as Chevron } from 'images/svg/careers/carousel-chevron.svg'
 
+type StyledButtonWrapperProps = {
+    disabled?: boolean
+    is_reviews?: boolean
+    left?: boolean
+}
+
+type NavigationContainerType = {
+    navigation_css?: FlattenSimpleInterpolation
+    bottom_offset?: string
+    height?: string
+}
+
+type ChevronRightAndLeftProps = {
```

Comment:
does this need `Type` in the end ?

Reply:
No @ali-hosseini-fs as I checked they are using one of these 2 conventions: ComponentNameProps || ComponentNameTypes